src/content/docs/en/guides/deploy/zeabur.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Deploy your Astro Site to Zeabur
+description: How to deploy your Astro site to the web on Zeabur.
+type: deploy
+i18nReady: true
+---
+[Zeabur](https://zeabur.com) offers hosting for fullstack web applications. Any Astro site can be hosted on Zeabur as SSR or static site.
+
+This guide includes instructions for deploying to Zeabur through the website UI.
+
+## Project Configuration
+
+Your Astro project can be deployed to Zeabur in two different ways: as a static site, or a server-rendered site.
+
+### Static Site
+
+Your Astro project is a static site by default. You don’t need any extra configuration to deploy a static Astro site to Zeabur. 
+
+### Server-Rendered Site
+
+1. Add the following to your `astro.config.mjs` file:
```

Comment:
I would suggest adding a step before `1.` about adding the `@astrojs/node` SSR adapter.

Have a look the [AWS Flightcontrol](https://docs-git-fork-michaelyuhe-main-astrodotbuild.vercel.app/en/guides/deploy/flightcontrol/) deployment recipe to see what I mean :)

Reply:
Thank you, I will add the step :)